package.json

```diff
@@ -9,16 +9,15 @@
     "hyperswarm-dht": "./dht.js"
   },
   "dependencies": {
-    "@hyperswarm/dht": "^3.6.2",
-    "@hyperswarm/discovery": "^1.11.4",
-    "dht-size-up": "^1.0.0",
+    "@hyperswarm/dht": "^4.0.1",
+    "@hyperswarm/discovery": "^2.0.1",
     "hyperswarm": "^2.13.0",
     "minimist": "^1.2.5",
     "pump": "^3.0.0",
-    "sodium-native": "^3.1.1"
+    "sodium-universal": "^3.0.4"
```

Comment:
it uses native in the code actually

Reply:
Sorry, I used the wrong cli command order 😅 . Fixed & force-pushed.